fleetd/fleetd.go

```diff
@@ -81,6 +81,7 @@ func main() {
 	cfgset.String("metadata", "", "List of key-value metadata to assign to the fleet machine")
 	cfgset.String("agent_ttl", agent.DefaultTTL, "TTL in seconds of fleet machine state in etcd")
 	cfgset.String("units_directory", "/run/fleet/units/", "Path to the fleet units directory")
+	cfgset.Bool("systemd_system", true, "When false use systemd --user)")
```

Comment:
Why not the other way around? I hope it's not just me -- but this looks kinda backwards...